<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器 2022414040107 单欣彤</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
    background-color: rgb(255, 230, 249);
	font: 10pt &quot;Arial Unicode MS&quot;;
    font-size:10pt;
}

QPushButton::hover{
	background-color: rgb(229, 252, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed{
	background-color: rgb(221, 222, 255);
	color: rgb(0, 0, 0);
}

QLineEdit{
font-size:35pt;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>342</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="display">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial Unicode MS</family>
             <pointsize>35</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnMS">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
             </property>
             <property name="text">
              <string>MS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMR">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
             </property>
             <property name="text">
              <string>MR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMC">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
             </property>
             <property name="text">
              <string>MC</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnPercentage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnClearAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnClear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnDel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>⌫</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnInverse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>1/x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnSquare">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>x^2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnSqrt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>√</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnDivide">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>÷</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnNum7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnNum8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnNum9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnMultipe">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btnNum4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btnNum5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btnNum6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnMinus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnNum1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btnNum2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btnNum3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btnPlus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnPlusMinus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>±</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btnNum0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="btnPeriod">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btnEqual">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>≡</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>计算器
</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="scienceShow"/>
     <addaction name="mainShow"/>
     <addaction name="action_6"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>换算</string>
     </property>
     <addaction name="action_9"/>
     <addaction name="action_10"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>设置</string>
     </property>
     <addaction name="action_13"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBiaozhun">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/new/prefix1/biaozhun.png</normaloff>:/new/prefix1/biaozhun.png</iconset>
   </property>
   <property name="text">
    <string>标准</string>
   </property>
   <property name="toolTip">
    <string>标准计算器</string>
   </property>
  </action>
  <action name="actionDaohang">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/new/prefix1/daohang.png</normaloff>:/new/prefix1/daohang.png</iconset>
   </property>
   <property name="text">
    <string>导航栏</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>标准

</string>
   </property>
  </action>
  <action name="scienceShow">
   <property name="text">
    <string>科学</string>
   </property>
  </action>
  <action name="mainShow">
   <property name="text">
    <string>程序员</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>日期计算</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>货币</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>容量</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>模式转换</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
  <include location="images/images.qrc"/>
  <include location="images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>scienceShow</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
